kpi/utils/django_orm_helper.py

```diff
@@ -77,10 +77,36 @@ def __init__(
         super().__init__(expression, order_list=order_list, **extra)
 
 
-class ReplaceValues(Func):
+class RemoveJSONFieldAttribute(Func):
+
+    """
+    Remove attribute from models.JSONField. It supports nested attributes by
+    targeting the attribute with its dotted path.
+    E.g., to remove `foo1` in `{"foo": {"foo1": "bar1", "foo2": "bar2"}}`,
+    `foo.foo1` should be passed as `attribute_dotted_path` parameter.
+    """
+
+    arg_joiner = ' #- '
+    template = "%(expressions)s"
+    arity = 2
+
+    def __init__(
+        self,
+        expression: str,
+        attribute_dotted_path: str,
+        **extra,
+    ):
+        super().__init__(
+            expression,
+            Value(f"{{{attribute_dotted_path.replace('.', ',')}}}"),
```

Comment:
This is kind of nasty. The combination of a function call (with arguments) inside a `f` string and all the curly braces gets to be a bit much.

Good ol' `+` is not so bad:
```python
Value('{' + attribute_dotted_path.replace('.', ',') + '}'),
```

Do I care strongly about this particular line of code? No. Do I want to avoid a slavish dedication to `f` strings as the one-and-only tool for string manipulation? Yes.